scripts/tests/test_zkai_d64_rmsnorm_to_projection_bridge_input.py

```
@@ -0,0 +1,68 @@
+    def test_payload_rejects_bridge_row_equality_drift(self) -> None:
+        payload = BRIDGE.build_payload()
+        row0 = payload["rows"][0]
+        base = row0["rmsnorm_normed_q8"]
+        row0["projection_input_q8"] = base + 1 if base < 127 else base - 1
+        with self.assertRaisesRegex(BRIDGE.BridgeInputError, "bridge row equality drift"):
+            BRIDGE.validate_payload(payload)
+
+    def test_payload_rejects_source_commitment_drift(self) -> None:
+        payload = BRIDGE.build_payload()
+        payload["source_rmsnorm_output_row_commitment"] = "blake2b-256:" + "77" * 32
+        with self.assertRaisesRegex(BRIDGE.BridgeInputError, "source_rmsnorm_output_row_commitment"):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)